### PR TITLE
CSS changes

### PR DESCRIPTION
**Pull request by coveros-matt-taylor**
Flattening things and adjusting colors, organizing the CSS somewhat more hierarchical...

Before:
image

image
_mobile view_

New:
image

image
_mobile view_

### CODE REVIEW

File: src/main/kotlin/coverosR3z/authentication/RegisterAPI.kt [hunk 1]
```diff
@@ -40,135 +40,117 @@ fun handlePOSTRegister(au: IAuthenticationUtilities, user: User, data: Map<Strin
...
 """+employees.joinToString("") { "<option value =\"${it.id.value}\">${it.name.value}</option>\n" } +
             """
-            <option selected disabled hidden>Choose here</option>
-          </select>
-          <button id="register_button" class="submit" align="center">Register</button>
-          </form>
-        </div>
-    </body>
-</html>
+          <option value ="1">Administrator</option>
```
Comment (7ep) [inline src/main/kotlin/coverosR3z/authentication/RegisterAPI.kt]: Was this line left intentionally?

**Review by 7ep — APPROVED** [review]
Woo woooooooooooo

File: src/main/kotlin/coverosR3z/authentication/RegisterAPI.kt (hunk 1 above)
Comment (coveros-matt-taylor) [reply src/main/kotlin/coverosR3z/authentication/RegisterAPI.kt]: Nope, probably should not be

Comment (coveros-matt-taylor) [reply src/main/kotlin/coverosR3z/authentication/RegisterAPI.kt]: Fixed

**Pull request merged**